### PR TITLE
#123 Remove auto updates for core

### LINKED ISSUE #123: Set WP_AUTO_UPDATE_CORE to false
We've removed the updates capability but not set the related constants which leads to failed attempts and error logs.

### PR DESCRIPTION
**Pull request by kirstyburgoine**
Uses `auto_update_core` filter

Fixes #123

### CODE REVIEW

**Comment by roborourke** [comment]
@kirstyburgoine I can't seem to find where this filter is used in WP, searching the codebase just takes me to the `WP_AUTO_UPDATE_CORE` constant

**Comment by kirstyburgoine** [comment]
@roborourke Sorry, this should have had WIP on it still. I found the filter here: https://wordpress.org/support/article/configuring-automatic-background-updates/#core-updates-via-filter but I haven't tested locally yet

**Comment by roborourke** [comment]
Hm, I think those docs are wrong or maybe for a future update :/ I can see the specific update type filters but not that global one. The constant is definitely present though

**Comment by roborourke** [comment]
Lets define this as false too `AUTOMATIC_UPDATER_DISABLED`

**Comment by kirstyburgoine** [comment]
@roborourke I've updated this to use `AUTOMATIC_UPDATER_DISABLED` now, I haven't also added `WP_AUTO_UPDATE_CORE` as per the original ticket because according to this doc: https://wordpress.org/support/article/configuring-automatic-background-updates/ `AUTOMATIC_UPDATER_DISABLED` will automatically disable all updates including core updates so it doesn't seem to be needed

File: inc/remove_updates/namespace.php [hunk 1]
```diff
@@ -8,6 +8,7 @@
 function bootstrap() {
 	add_action( 'admin_init', __NAMESPACE__ . '\\remove_maybe_update_checks', 1 );
 	add_action( 'admin_init', __NAMESPACE__ . '\\remove_update_nag' );
+	add_action( 'admin_init', __NAMESPACE__ . '\\remove_auto_updates' );
```
Comment (roborourke) [inline inc/remove_updates/namespace.php]: Last thing to do here - I don't think `admin_init` will always be fired in the context of upgrades / installs so you can call the function directly from here, it's fine as we're just defining a constant so earlier the better.

File: inc/remove_updates/namespace.php [hunk 2]
```diff
@@ -38,6 +39,14 @@ function remove_update_nag() {
 	remove_filter( 'update_footer', 'core_update_footer' );
 }
 
+/**
+ * Remove all automatic updates.
+ *
```
Comment (roborourke) [inline inc/remove_updates/namespace.php]:
```suggestion
```

**Pull request merged**